guides/en/applicationcfc.md

```diff
@@ -0,0 +1,87 @@
+| [onRequestStart][5]     | First page-processing function run when a page request starts. Return False to prevent ColdFusion from processing the request.                                                                                                                                           |
+| [onRequest][6]          | Runs when a request starts, after the onRequestStart event handler. This method is optional. If you implement this method, it must explicitly call the requested page to process it.                                                                                     |
+| [onCFCRequest][7]       | Intercepts any HTTP or AMF calls to an application based on CFC request. Whereas onRequest handles only requests made to ColdFusion templates, this function controls Ajax, Web Service, and Flash Remoting requests.                                                    |
+| [onRequestEnd][8]       | Runs at the end of a request, after all other CFML code.                                                                                                                                                                                                                 |
+| [onAbort][9]            | Runs when you execute the CFML tag cfabort or cfscript "abort". If showError attribute is specified in cfabort, onError method is executed instead of onAbort. When using cfabort, cflocation, or cfcontent tags, the onAbort method is invoked instead on onRequestEnd. |
+| [onError][10]           | Runs when an uncaught exception occurs in the application. This method overrides any error handlers that you set in the ColdFusion Administrator or in cferror tags. It does not override try/catch blocks.                                                              |
+| [onMissingTemplate][11] | Runs when a request specifies a non-existent CFML page. True, or no return value, specifies that the event has been processed. If the function returns false, ColdFusion invokes the standard error handler.                                                             |
+
+[1]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onApplicationStart
+[2]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onApplicationEnd
+[3]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onSessionStart
+[4]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onSessionEnd
+[5]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onRequestStart
+[6]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onRequest
+[7]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onCFCRequest
+[8]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onRequestEnd
+[9]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onAbort
+[10]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onError
+[11]: https://wikidocs.adobe.com/wiki/display/coldfusionen/onMissingTemplate
```

Comment:
We _should_ have doc's for all these methods, should link locally. 

Also not sure if this needs to be split into a guide... is it more useful to have the example component on the `Application.cfc` doc instead of in a separate guide where it will be harder to find?

Reply:
There are more comments than code. This example was pretty huge. That did not look very well.

This methods should be emphasized as lifecycle hooks in my opinion...

Reply:
I think the idea of a `application` guide might make a lot of sense... It could explain what Application.cfc is, how it is looked for, talk about `Application.cfm` and `OnRequestEnd.cfm` (discouraged), etc. Do you want to rename it as `application.md` and we can cover all things CFML application and link to the various sub docs from there?

Reply:
That's a good point. There is already a guided for [application scope](https://cfdocs.org/application-scope). Should this one be used for that so both can be merged?

Reply:
Yeah I think application scope could be merged into the application guide.

Reply:
Okay, I'll do so :+1: